developer-portal/for-developers/bigcommerce/headless/fast-checkout-on-amp-pages.md

```diff
@@ -0,0 +1,54 @@
+---
```

Comment:
this seems mostly copied from the `_TEMPLATE-PLATFORM_NAME-fast-checkout-on-amp-pages.md`. Any way to make it all shared?